include/osquery/tables.h

```diff
@@ -67,7 +67,7 @@ namespace tables {
 
 /// Helper alias for TablePlugin names.
 typedef const std::string TableName;
-typedef const std::vector<std::pair<std::string, std::string> > TableColumns;
+typedef std::vector<std::pair<std::string, std::string> > TableColumns;
```

Comment:
Thanks! Sorry for dropping off from the review after leaving a few lines of notes. I was distracted by desk conversations. 

Removing the const in the typedef is definitely the way to go!